src/DGtal/geometry/volumes/ConvexityHelper.h

```diff
@@ -0,0 +1,424 @@
+/**
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+
+#pragma once
+
+/**
+ * @file ConvexityHelper.h
+ * @author Jacques-Olivier Lachaud (\c jacques-olivier.lachaud@univ-savoie.fr )
+ * Laboratory of Mathematics (CNRS, UMR 5127), University of Savoie, France
+ *
+ * @date 2020/12/20
+ *
+ * Header file for module ConvexityHelper.cpp
+ *
+ * This file is part of the DGtal library.
+ */
+
+#if defined(ConvexityHelper_RECURSES)
+#error Recursive header files inclusion detected in ConvexityHelper.h
+#else // defined(ConvexityHelper_RECURSES)
+/** Prevents recursive inclusion of headers. */
+#define ConvexityHelper_RECURSES
+
+#if !defined ConvexityHelper_h
+/** Prevents repeated inclusion of headers. */
+#define ConvexityHelper_h
+
+//////////////////////////////////////////////////////////////////////////////
+// Inclusions
+#include <iostream>
+#include <string>
+#include <vector>
+#include "DGtal/base/Common.h"
+#include "DGtal/kernel/SpaceND.h"
+#include "DGtal/geometry/tools/QuickHull.h"
+#include "DGtal/geometry/volumes/BoundedLatticePolytope.h"
+#include "DGtal/geometry/volumes/BoundedRationalPolytope.h"
+#include "DGtal/geometry/volumes/ConvexCellComplex.h"
+#include "DGtal/shapes/PolygonalSurface.h"
+
+namespace DGtal
+{
+  /////////////////////////////////////////////////////////////////////////////
+  // template class ConvexityHelper
+  /**
+     Description of template class 'ConvexityHelper' <p> \brief Aim:
+     Provides a set of functions to facilitate the computation of
+     convex hulls and polytopes, as well as shortcuts to build cell
+     complex representing a Delaunay complex.
+
+     @tparam dim the dimension of the space where points and further objects live.
+  */
```

Comment:
missing doc for TInteger type (and this type is not "concept checked")

Reply:
done